### ISSUE TITLE
logout

### ISSUE THREAD

**Issue by mindscratch** [goji/httpauth#19]
How might I logout a user?

**Comment by elithrar** [comment]
HTTP Basic Authentication doesn’t have the concept of sessions: a client
must supply their credentials on every request.

If you want session-based authentication I would recommend a more robust
approach with database backed passwords and cookie or token sessions.

**Comment by mindscratch** [comment]
So does the browser cache something? When I access the site with basic auth, the first time I enter username/password, then it no longer prompts until I clear browser cache or use a "private" browser window.

**Comment by elithrar** [comment]
Yes, the browser caches it as a convenience, but the server has no control
over that.
